strapi/strapi
# (Front)Count draft relations on publish

### What does it do?

Display a warning when publishing an entity with draft relations

### How to test it?

- Attempt to publish an entity (single or collection) that has a number of draft relations. 
- You should see a warning before proceeding to publish

Relies on BE changes in #14547
